configure.ac

```diff
@@ -197,6 +197,20 @@ AM_COND_IF([USE_RADOS_AIORI],[
         AC_DEFINE([USE_RADOS_AIORI], [], [Build RADOS backend AIORI])
 ])
 
+# CEPHFS support
+AC_ARG_WITH([cephfs],
```

Comment:
In the future, the code might benefit from automatic detection of include/library files. It is reasonable to keep it function for now though.